### PR TITLE
CNS - spec add query info

### PR DESCRIPTION
**Pull request by omerlavanet**

### CODE REVIEW

File: proto/spec/query.proto [hunk 1]
```diff
@@ -87,6 +87,7 @@ message ShowAllChainsInfoStruct {
 	string chainName = 1;
 	string chainID = 2;
 	repeated string enabledApiInterfaces = 3;
+  uint64 api_count = 4;
```
Comment (Yaroms) [inline proto/spec/query.proto:90]: tab :)

File: x/spec/keeper/grpc_query_show_all_chains.go [hunk 2]
```diff
@@ -26,14 +26,26 @@ func (k Keeper) ShowAllChains(goCtx context.Context, req *types.QueryShowAllChai
 		chainName := spec.GetName()
 		chainId := spec.GetIndex()
 
+		expectedInterfaces := make(map[string]bool)
```
Comment (Yaroms) [inline x/spec/keeper/grpc_query_show_all_chains.go:29]: why do we need the make here?

Comment (omerlavanet) [reply x/spec/keeper/grpc_query_show_all_chains.go:29]: Because the internal function expects a map and updates it

Comment (Yaroms) [reply x/spec/keeper/grpc_query_show_all_chains.go:29]: but it receives an empty map and than you override the existing one' why not just create the map inside the method?

**Review by Yaroms — APPROVED** [review]

**Pull request merged**